### ISSUE TITLE
Giving an example for a calibration/validation set

### ISSUE THREAD

**Issue by SebastienMelo** [benchopt/template_benchmark_ml#1]
In the objective.py file, it would be useful to give an example of how to incorporate a calibration set on which you tune hyperparameters for instance.
ie. make more explicit how to handle calibration sets in solvers.
Thanks!
